skype/skype-ps/skype/Get-CsOnlineDirectoryTenant.md

```diff
@@ -22,7 +22,8 @@ Get-CsOnlineDirectoryTenant [[-Tenant] <Guid>] [-DomainController <Fqdn>] [-Forc
 ```
 
 ## DESCRIPTION
-**Note**: Starting with Teams PowerShell Module 4.0 (available from March 1, 2022), this cmdlet is deprecated. Use the Get-CsTenant cmdlet to view information previously present in Get-CsOnlineDirectoryTenant.
+**Note**: Starting with Teams PowerShell Module 4.0, this cmdlet will be deprecated on 30th April 2022. Use the Get-CsTenant or  Get-CsOnlineDialInConferencingBridge cmdlet to view information previously present in Get-CsOnlineDirectoryTenant.
```

Comment:
```suggestion
**Note**: Starting with Teams PowerShell Module 4.0, this cmdlet will be deprecated. Use the Get-CsTenant or  Get-CsOnlineDialInConferencingBridge cmdlet to view information previously present in Get-CsOnlineDirectoryTenant.
```

Reply:
Microsoft Docs policy is to not publish specific future dates, so I would suggest removing the date.